src/test/app.test.js

```diff
@@ -0,0 +1,10 @@
+import App from '../components/app'
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+describe("Tests if all components are being rendered and work as expected", () => {
+    it("Whole app component should render correctly", () => { 
+        const div = document.createElement('div');
+        ReactDOM.render(<App />, div);
```

Comment:
There's no assertion here. You're not testing for anything.

Reply:
You can check again 
Corrected
Now I am testing for something.